blint/android.py

```diff
@@ -278,14 +289,21 @@ def parse_so_file(app_file, app_temp_dir, sof):
                 field=FieldModel.purl,
                 confidence=0.5,
                 methods=[
-                    Method(technique=Technique.binary_analysis, value=rel_path, confidence=0.5, )
+                    Method(
+                        technique=Technique.binary_analysis,
+                        value=rel_path,
+                        confidence=0.5,
+                    )
                 ],
             )
         ),
         properties=[
             Property(name="internal:srcFile", value=rel_path),
             Property(name="internal:appFile", value=app_file),
-            Property(name="internal:functions", value=", ".join(set(functions)), ),
+            Property(
```

Comment:
@prabhu This formatting seems a bit random, no?

Reply:
I ran black. How do you format?

Reply:
@prabhu I've been avoiding black lately.

What I mean is that you look at lines 352-356 of the same file, they are much longer. Given the other Property calls are all on one line, there is no reason to chop up just this one.

Current
![image](https://github.com/owasp-dep-scan/blint/assets/80227828/f47f44cb-d9f2-4075-a067-cd0f38675449)

On one line
![image](https://github.com/owasp-dep-scan/blint/assets/80227828/67eccbd4-ccb9-4b20-8610-14d90cf384e1)

Reply:
Changed to 1 line